test/exampleAnnotations.yaml

```diff
@@ -0,0 +1,4 @@
+pachd:
+  serviceAccount:
+    additionalAnnotations:
+      eks.amazonaws.com/role-arn: blah123
```

Comment:
I think your editor doesn’t insert newlines at the end of files.

Reply:
I like `annotations` better as well, but used `additionalAnnotations` because we insert our own annotations as well.